deploy/olm-catalog/ibm-iam-operator/4.4.0/ibm-iam-operator.v4.4.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,784 @@
+apiVersion: operators.coreos.com/v1alpha1
```

Comment:
@Tirumalavasa @ujjwalchk-it CSV should be 4.4.1?

Reply:
@rashmi43 , we also have 4.4.1 CSV